docs/downloads.md

```diff
@@ -6,4 +6,12 @@ sidebar_position: 100
 
 Right now all `zrok` binaries are delivered from the release page at GitHub.
 
-Head over to https://github.com/openziti/zrok/releases/latest to find the latest downloads.
\ No newline at end of file
+Head over to https://github.com/openziti/zrok/releases/latest to find the latest downloads.
+
+You need the right download for your OS and CPU. This can be confusing for Raspberry Pi users because their Raspberry Pi OS could be 32bit or 64bit. Linux users can find the right version by looking up the result of `uname -m` in this table.

```

Comment:
if you're going to have me run `uname -m`, the lead value should be the value i see, it shouldn't be buried in the parenthesis imo... also the table needs a heading... "Result" maybe?

|                             |Linux Download Name |
|-----------------------------|------------|
| x86_64          |linux-amd64 |
| aarch64, arm/v8 |linux-arm64 |
| armhf, arm/v7   |linux-armv7 |